src/main/java/org/swisspush/redisques/QueueStatsService.java

```diff
@@ -209,13 +213,13 @@ private int compareLargestFirst(Queue a, Queue b) {
     }
 
 
-    private static class GetQueueStatsRequest<CTX> {
+    static class GetQueueStatsRequest<CTX> {
```

Comment:
Do not change visibility of methods/variables just to test them. This is bad practice. If you can find a workaround or there is a arguable reason, it may be ok.